src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,231 @@
+        }
+    }
+
+    /**
+     * The method returns all data from the table.
+     *
+     * @param tableName the name of the table whose data should be returned
+     * @return all data from the table in the format Field1 | Field2 | ... | FieldN
+     * @throws SQLException if a database access error occurs
+     */
+    private String getTableData(String tableName) throws SQLException {
+        String query = "SELECT * FROM \"" + tableName + "\"";
+        try (Statement statement = connection.createStatement(); ResultSet resultSet = statement.executeQuery(query)) {
+            int columnCount = resultSet.getMetaData().getColumnCount();
+            StringBuilder result = new StringBuilder();
+            while (resultSet.next()) {
+                for (int i = 1; i <= columnCount; i++) {
+                    result.append(resultSet.getString(i)).append(" | ");
+                }
+                result.replace(result.length() - 3, result.length(), System.lineSeparator());
+            }
+            return result.toString();
+        }
+    }
+
+    /**
+     * Method adds users to the table "user".
+     *
+     * @param users array of users and their passwords.
+     *              One element of array it is one user and his password.
+     *              One element of array has format username|password.
+     * @throws SQLException if a database access error occurs
+     */
+    private void addUsers(String[] users) throws SQLException {
+        if (users == null) {
+            System.err.println("List of users is null.");
+        } else {
+            for (String oneUser : users) {
+                if (oneUser != null) {
+                    addOneUser(oneUser.split("\\|"));
+                } else {
+                    System.err.println("Can't add some user. Name and password are Null");
+                }
+            }
+        }
+
+    }
+
+    /**
+     * Method adds one user in the table "user".
+     *
+     * @param oneUserNameAndPassword array with two element.
+     *                               Element with index 0 it is the userneme.
+     *                               Element with index 1 it is the password.
+     * @throws SQLException if a database access error occurs
+     */
+    private void addOneUser(String[] oneUserNameAndPassword) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("INSERT INTO \"user\" (name, password) VALUES ('%s', '%s')",
+                    oneUserNameAndPassword[0], oneUserNameAndPassword[1]);
+            statement.executeUpdate(query);
+        }
+    }
+
+    /**
+     * Method creates the new table.
+     *
+     * @param tableName the name of the table that the method creates.
+     * @throws SQLException if a database access error occurs
+     */
+    private void createTable(String tableName) throws SQLException {
+        if (!isTableExist(tableName)) {
```

Comment:
I think you should use sql syntax instead isTableExist

link - https://stackoverflow.com/questions/1766046/postgresql-create-table-if-not-exists

Reply:
I will do it. 
Thanks for link.